### ISSUE TITLE
Feature Request: Include a list of file extensions that are known to be compressed to be excluded from compression

### ISSUE THREAD

**Issue by ptandler** [kopia/kopia#2190]
I'd assume that in most cases it doesn't gain a lot when a compressed file is compressed again when storing to the repository.

So, I think it would be helpful to ease setup, to include a list of file extensions that are known to be compressed - in such a way that they can be easily excluded from compression by a policy.

Actually, I'm not sure what would be the best way to do this. Ideas:

- store this list internally and just include another boolean flag in the policy. Hm, maybe not preferred as it is not obvious what will be excluded and the list cannot be tweaked to custom needs
- Kopia UI could include a button to add these extensions to a policy in the editor - but no idea how this could be done for the CLI version

Anyway, this is the list I'm currently using:

```
jpg
jpeg
png
svgz
mp3
m4a
ogg
avi
mov
mp4
m4v
mpg
mpeg
ogv
7z
zip
gz
xz
zst
zstd
bz2
tgz
rar
jar
war
webm
webp
docx
pptx
xlsx
```

**Comment by Spellshaper** [comment]
In my experience Kopia performance does not really improve when skipping already compressed data unless you choose a deliberately CPU-intensive compression setting, in which case i have observed that png, jpg and (non-encrypted) zip at least can benefit from a second round of compression.
I would also count docx, pptx and xlsx as zip because that's what they are - structured xml documents inside a zip archive with a different extension.
Choosing any flavor of S2 compression should already skip compression for blocks that do not benefit from it, if i'm reading the S2 docs right.

In conclusion, I am not sure a pre-included list like this would be of benefit on average, whether applied by default or not.

**Comment by ptandler** [comment]
Oh, there _is_ a benchmark command already! Oops, I've overseen this. Wow! Kopia is _really_ amazing! Thanks a lot!

I'll try benchmarking once with excluded files, once without ... when I've got some spare time.
